src/uct/api/addr.h

```diff
@@ -7,6 +7,7 @@
 #ifndef UCT_API_ADDR_H_
 #define UCT_API_ADDR_H_
 
+#include "config.h"
```

Comment:
Okay, I figured the solution would look like that. I was just in a bit of a bind because that struct needs to be there in some way. I'll move it to be smsg internal.